GoogleChromeLabs/wp-sitemaps
# Return a 404 when there are no URLs for a sitemap page.

### Issue Number
This fixes #57.

### Description
This consolidates all sitemap rendering logic in a single `render_sitemap()` method in the base `Core_Sitemaps_Provider` class and checks for an empty URL list before rendering the sitemap and returns a 404 if no URLs are found.

### Type of change
Please select the relevant options:


### Steps to test
1. Visit a sitemap page with a very large page number, like `/sitemap-posts-post-99999.xml`.
2. See that the theme shows a 404 page.

### Acceptance criteria
